lib/helpscout/api.rb

```diff
@@ -80,7 +90,7 @@ def cleansed_params(params)
     def client
       @client ||= Faraday.new(url: BASE_URL) do |conn|
         conn.request :json
-        conn.basic_auth(Helpscout.api_key, 'X')
+        conn.authorization(:Bearer, Helpscout.access_token.token) if Helpscout.access_token
```

Comment:
If we somehow get here and there is no `access_token` should we request a new one?